readthedocs/templates/projects/project_redirects.html

```diff
@@ -22,8 +22,12 @@
                 redirect_target = "/$lang/$version/faq.html";
             }
             else if (option === "page" ) {
-                redirect_from = "/$lang/$version/" + field_from_url.val();
-                redirect_target = "/$lang/$version/" + field_to_url.val();
+                field_from = field_from_url.val();
+                field_to = field_to_url.val();
+                field_from_absolute = (field_from.length > 0) && (field_from[0] === "/");
+                field_to_absolute = (field_to.length > 0) && (field_to[0] === "/");
+                redirect_from = "/$lang/$version" + (field_from_absolute ? "" : "/") + field_from;
+                redirect_target = "/$lang/$version" + (field_to_absolute ? "" : "/") + field_to;
```

Comment:
Oh. yes, that's much cleaner, I'll edit the PR accordingly. Thank you for the review.